app/controllers/mixins/ems_common_angular.rb

```diff
@@ -116,6 +116,7 @@ def create
 
     def get_task_args(ems)
       user, password = params[:default_userid], MiqPassword.encrypt(params[:default_password])
+      metrics_user, metrics_password = params[:metrics_userid], MiqPassword.encrypt(params[:metrics_password])
```

Comment:
Do any other providers use `metrics_` auth type?  We might want to move these provider specific ones down into the respective case statements otherwise this list can grow quickly (add hawkular and prometheus for containers at least)